### PR TITLE
Add documentation for omisego-admin

### LINKED ISSUE #69: Add code documentation for `omisego-admin`
Currently, `omisego-admin` module doesn't have documentation. We want a documentation for public functions to help developer easier to understand how to use the sdk.

### PR DESCRIPTION
**Pull request by ripzery**
Issue/Task Number: #69 

Closes #69 

# Overview

This PR adds a documentation for `omisego-admin` module.

### CODE REVIEW

File: omisego-admin/README.md [hunk 1]
```diff
@@ -1 +1,264 @@
...
+})
+```
+
+There are different ways to initialize a `TransactionCreateParams` by specifying either `toAddress`, `toAccountId` or `toProviderUserId`.
+
+### PaginationList
+`PaginationList` is an object represents a paginated filtered data set. 
+This object will be returned from any APIs that probably contain multiple objects (e.g. account list, transaction list, token list, etc.).
+
+The general structure of the paginated request parameters is consist of:
```
Comment (T-Dnzt) [inline omisego-admin/README.md]: `request parameters consists of`

File: omisego-admin/README.md [hunk 2]
```diff
@@ -1 +1,264 @@
...
+    override fun fail(response: OMGResponse<APIError>) {
+        // Handle error
+    }
+})
+```
+
+There are different ways to initialize a `TransactionCreateParams` by specifying either `toAddress`, `toAccountId` or `toProviderUserId`.
+
+### PaginationList
+`PaginationList` is an object represents a paginated filtered data set.
```
Comment (T-Dnzt) [inline omisego-admin/README.md]: `is an object representing a paginated filtered data set`

File: omisego-admin/README.md [hunk 3]
```diff
@@ -1 +1,264 @@
...
+        
+    }
+
+    override fun success(response: OMGResponse<AuthenticationToken>) {
+        
+    }
+})
+```
+
+> Note: The `Authentication` header will be set automatically when login successfully.
```
Comment (T-Dnzt) [inline omisego-admin/README.md]: `when logging in successfully`

File: omisego-admin/README.md [hunk 4]
```diff
@@ -1 +1,264 @@
...
+        // Handle error
+    }
+})
+```
+
+There are different ways to initialize a `TransactionCreateParams` by specifying either `toAddress`, `toAccountId` or `toProviderUserId`.
+
+### PaginationList
+`PaginationList` is an object represents a paginated filtered data set. 
+This object will be returned from any APIs that probably contain multiple objects (e.g. account list, transaction list, token list, etc.).
```
Comment (T-Dnzt) [inline omisego-admin/README.md]: `This object will be returned from any API listing resources (e.g. accounts, transactions, tokens, etc.).

File: omisego-admin/README.md [hunk 5]
```diff
@@ -1 +1,264 @@
...
+
+The general structure of the paginated request parameters is consist of:
+
+* `page` is the page you wish to receive
+* `perPage` is the number of results per page
+* `sortBy` is the available sorting fields. This value depends on which API is being requested.
```
Comment (T-Dnzt) [inline omisego-admin/README.md]: `contains the field used to sort. The value depends on which endpoint is being used.`

File: omisego-admin/README.md [hunk 6]
```diff
@@ -1 +1,264 @@
...
+* `sortDir` is the available sorting direction. Available sorting directions are:
```
Comment (T-Dnzt) [inline omisego-admin/README.md]: `contains the direction for sorting. Available sorting directions are:`

File: omisego-admin/README.md [hunk 7]
```diff
@@ -1 +1,264 @@
...
+    
+    `ASCENDING`, `DESCENDING`
+    
+    > `import co.omisego.omisego.model.pagination.SortDirection.*`
+    
+* `searchTerm` *(optional)* is a term to search for all of the searchable fields.
```
Comment (T-Dnzt) [inline omisego-admin/README.md]: `will search in all the searchable fields with a `OR` mechanism.

File: omisego-admin/README.md [hunk 8]
```diff
@@ -1 +1,264 @@
...
+        //TODO: Do something with the paginated list of transactions
+    }
+})
+```
+   
+#### Get account list
+
+This returns a paginated filtered list of `accounts`.
+
+In order to get the `account list` you will need to create a `AccountListParams` object in the similar way to [Get transaction list](#get-transaction-list)
```
Comment (T-Dnzt) [inline omisego-admin/README.md]: `in a similar way to`

**Comment by ripzery** [comment]
@T-Dnzt Fixed wording 😄

**Pull request merged**